walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -4,13 +4,13 @@
     <meta charset="UTF-8">
     <title>Random Quotes</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script src="script1.js"></script>
```

Comment:
Is this correct? I don't see a `script1.js` file checked into this repository.

Reply:
Must've missed it in my last push. It should be there now. Thank you!